docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +39,324 @@ rg.init(workspace="my_shared_workspace")
+#  'api_key': 'SrX9T_4DAWK65Ztp4sADfB3g05t3bpwjwgfIwR3BP90uRg_LkWlsBXccAI9KTRbedxMNDdw15pM-9p56vPQhFv88d8e-M7PzVDZave92qPA',
+#  'inserted_at': '2023-03-16T11:17:35.462774',
+#  'updated_at': '2023-03-16T11:17:35.462774'}]
+````
+
+## Delete an user
 
-To configure your Argilla instance for various users, you just need to create a yaml file as follows:
-```yaml
-#.users.yaml
-# Users are provided as a list
-- username: user1
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser1APIKEY"
-  workspaces: [] # This user will only have her user workspace available
-- username: user2
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser2APIKEY"
-  workspaces: ['client_projects'] # access to her user workspace and the client_projects workspace
-- username: user3
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser2APIKEY" # this user can access all workspaces
-- ...
-```
-
-Then point the following environment variable to this yaml file before launching the server:
+```python
+http.delete("/api/users/75190fff-d4b9-4625-b7d3-4cfe3c659054").json()
+# {'id': '75190fff-d4b9-4625-b7d3-4cfe3c659054',
+# 'username': 'nick2',
+# 'role': 'annotator',
+# 'full_name': 'Nick Name',
+# 'workspaces': ['ws2'],
+# 'api_key': 'SrX9T_4DAWK65Ztp4sADfB3g05t3bpwjwgfIwR3BP90uRg_LkWlsBXccAI9KTRbedxMNDdw15pM',
+# 'inserted_at': '2023-03-16T11:17:35.462774',
+# 'updated_at': '2023-03-16T11:17:35.462774'}
 
-```bash
-export ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/path/to/.users.yaml
 ```
 
-If everything went well, the configured users can now log in and their annotations will be tracked with their usernames.
 
+## HOWTO migrate users from the `users.yaml` file
+
+Given the following users yaml file
+````yaml
+- username: john
+  full_name: John Doe
+  email: john@argilla.io
+  api_key: a14427ea-9197-11ec-b909-0242ac120002
+  hashed_password: $2y$05$xtl7iy3bpqchUwiQMjEHe.tY7OaIjDrg43W3TB4EHQ7izvdjvGtPS
+  disabled: False
+
+- username: tanya
+  full_name: Tanya Franklin
+  email: tanya@argilla.io
+  api_key: 78a10b53-8db7-4ab5-9e9e-fbd4b7e76551
+  hashed_password: $2y$05$aqNyXcXRXddNj5toZwT0HugHqKZypvqlBAkZviAGGbsAC8oTj/P5K
+  workspaces: [argilla, team]
+  disabled: True
+
+- username: daisy
+  full_name: Daisy Gonzalez
+  email: daisy@argilla.io
+  api_key: a8168929-8668-494c-b7a5-98cd35740d9b
+  hashed_password: $2y$05$l83IhUs4ZDaxsgZ/P12FO.RFTi2wKQ2AxMK2vYtLx//yKramuCcZG
+  workspaces: [argilla, team, latam]
+  disabled: False
+
+````
+
+The migration tasks will create users and workspaces properly.
+
+The user role will be computed depending
+on how workspaces are setup for each user. If no `workspace` attribute is defined, user will be
+considered as an `admin`. Otherwise, the assigned user role will be `annotator`. Also, and for each user
+too, one extra workspace with the same name as the username will be created.
```

Comment:
```suggestion
The user role will be computed depending on how workspaces are setup for each user. If no `workspace` attribute is defined, the user will be considered an `admin`. Otherwise, the assigned user role will be `annotator`. 

The task will also create an extra workspace for each user named after their username.
```